packages/server/modules/notifications/helpers/types.ts

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { MaybeAsync, Optional } from '@/modules/shared/helpers/typeHelper'
+import { Job } from 'bull'
+import debug from 'debug'
+import { isObject, has } from 'lodash'
+
+export enum NotificationType {
+  MentionedInComment = 'mentioned-in-comment',
+  Test = 'test'
```

Comment:
Can we not add the `Test` case in the production code? Maybe make this an extendable type and extend in the tests.

Reply:
got rid of it everywhere. this does make the test-push and test-consume commands of the CLI a bit confusing, because I'm having to use the real "Mentioned in Comment" notification type to test out pushing & consuming, while overriding its handler to be a "test" handler